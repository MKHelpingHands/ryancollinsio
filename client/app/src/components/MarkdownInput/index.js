import React, { PropTypes } from 'react';
import styles from './index.module.scss';
import cssModules from 'react-css-modules';

const MarkdownInput = ({
  field,
}) => (
  <textarea rows={6} columns={40} {...field} />
);

MarkdownInput.propTypes = {
  field: PropTypes.object.isRequired,
};

export default cssModules(MarkdownInput, styles);
