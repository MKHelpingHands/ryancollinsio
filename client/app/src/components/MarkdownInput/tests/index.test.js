import MarkdownInput from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<MarkdownInput />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <MarkdownInput />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
