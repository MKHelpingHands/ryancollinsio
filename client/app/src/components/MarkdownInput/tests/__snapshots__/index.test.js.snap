exports[`<MarkdownInput /> should render with default props 1`] = `
<textarea
  columns={40}
  rows={6} />
`;
